app/assets/javascripts/initializers/initializeRuntimeBanner.js

```diff
@@ -0,0 +1,62 @@
+/* global Runtime */
+
+function handleDismissRuntimeBanner(event) {
```

Comment:
These could also live/get imported into base.jsx then they are in an ES module not polluting the global.

Reply:
By making them "live/get imported into base.jsx" do you mean they would be added to `window.Forem.handleDismissRuntimeBanner`? Or just placing them inside that file without a namespace (like [`initializeNav()`](https://github.com/forem/forem/blob/main/app/javascript/packs/base.jsx#L92))?

I'm thinking that in a way it makes sense to keep them close to the rest of the logic here, otherwise when revisiting this code we would have to go looking in multiple files for these functions. It's not a strong opinion that I want to defend though, just looking for confirmation on this before moving them.

Reply:
You could do it like `initializeNav()` unless there is some other reason it needs to be exposed on the global object.

Reply:
I did my best attempt at moving it there but fell into an InstantClick edge case(s) and wasn't able to get out of it (some Cypress tests started failing in some cases). Keeping it here helps a lot because it's ensured to run on all new pages (including InstantClick navigations).

If we really want to move it to `base.jsx` it would be great if we can make that a separate effort to avoid blocking this PR for longer. Otherwise I'll probably need help sorting out a good way to move the logic there.

Reply:
@nickytonline please let me know if this sounds alright with you whenever you have a chance. Thanks!

Reply:
Sounds good @fdoxyz. We can defer moving things to base.jsx for now.